### ISSUE TITLE
logical 'and' results in unexpected char & error

### ISSUE THREAD

**Issue by robertpdx** [elapouya/python-docx-template#71]
I'm getting an error with the following conditional using a logical and:
~~~~
{% if loop.index0 % 5==0 and loop.index0 > 0 %}
~~~~

The last line of the trace is:
~~~~
jinja2.exceptions.TemplateSyntaxError: unexpected char '&' at xxxxxxx
~~~~

Any ideas?  I tried grouping with parentheses but it results in the same error.
docxtpl is version 0.3.4 and Jinja2 is version 2.9.5

**Comment by robertpdx** [comment]
Unable to reproduce in the test files, it must be an issue with my particular template.

**Comment by elapouya** [comment]
Please, could you attach your .docx template

**Comment by robertpdx** [comment]
Here's a modified template using the nested for loop example from the test files.
[nested_for_tpl.docx](https://github.com/elapouya/python-docx-template/files/786262/nested_for_tpl.docx)

**Comment by elapouya** [comment]
There was a bug into one regular expression : I corrected it in docxtpl 0.3.5 I just pushed

**Comment by robertpdx** [comment]
Thank you for the quick fix, that took care of it.
